common/oatbox/cache/ItemPoolSimpleCacheAdapter.php

```diff
@@ -132,6 +134,10 @@ public function commit()
 
     private function getCache(): CacheInterface
     {
+        if ($this->hasOption(self::OPTION_CACHE_SERVICE)) {
```

Comment:
What do u think about putting this GCPCache in default configuration (in place of SimpleCache::SERVICE_ID) instead of hijacking the dependency here?
It will be set only on GCP environment ofc

Reply:
You mean convert `ItemPoolSimpleCacheAdapter.php` to a configurable service, create a `.conf.php` and add it in the migrations / installation?

What is the difference between this? It is not hijacking. This is the tao DI, I can set options when I build service. Did not see the problem doing it.

Reply:
i don't see any benefit of doing the change